addons/project/views/project_views.xml

```diff
@@ -649,9 +649,9 @@
                     <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
                     <div class="oe_title pr-0">
                         <h1 class="d-flex flex-row justify-content-between">
-                            <field name="priority" widget="priority" class="mr-3"/>
+                            <field name="priority" widget="priority" class="mr-3" groups="project.group_project_user"/>
```

Comment:
We could do that:
```suggestion
                            <field name="priority" widget="priority" class="mr-3" groups="project.group_project_user"/>
                            <field name="priority" widget="priority" class="mr-3" readonly="1" groups="!project.group_project_user"/>
```
to make a field readonly when the user has no access to project app.

Reply:
hello @xavierbol ,

The changes have been implemented. Thank you for the suggestion.slightly_smiling_face